examples/bpk-component-autosuggestV2/examples.js

```diff
@@ -0,0 +1,243 @@
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import { Component, ReactNode } from 'react';
+
+import { withRtlSupport } from '../../packages/bpk-component-icon';
+import FlightIcon from '../../packages/bpk-component-icon/lg/flight';
+import BpkAutosuggest, {
+  BpkAutosuggestSuggestion,
+} from '../../packages/bpk-component-autosuggestV2';
+
+const BpkFlightIcon = withRtlSupport(FlightIcon);
+
+const offices = [
+  {
+    name: 'Barcelona',
+    code: 'BCN',
+    country: 'Spain',
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: 'Beijing',
+    code: 'Any',
+    country: 'China',
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: 'Budapest',
+    code: 'BUD',
+    country: 'Hungary',
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: 'Edinburgh',
+    code: 'EDI',
+    country: 'United Kingdom',
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: 'Glasgow',
+    code: 'Any',
+    country: 'United Kingdom',
+    indent: true,
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: 'London',
+    code: 'Any',
+    country: 'United Kingdom',
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: 'Miami, FL',
+    code: 'Any',
+    country: 'United States',
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: "Shenzhen Bao'an International",
+    code: 'SZX',
+    country: 'China',
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: 'Singapore Changi',
+    code: 'SIN',
+    country: 'Singapore',
+    tertiaryLabel: 'Tertiary label',
+  },
+  {
+    name: 'Sofia',
+    code: 'SOF',
+    country: 'Bulgaria',
+    tertiaryLabel: 'Tertiary label',
+  },
+];
+
+const dataHanzi = [
+  {
+    name: '深圳寶安國際 (Shenzhen)',
+    code: 'SZX',
+    country: '中國',
+    tertiaryLabel: '三級標籤',
+  },
+];
+
+const sections = [
+  {
+    title: 'Recent searches',
+    suggestions: [offices[0]],
+  },
+  {
+    title: 'Popular locations',
+    suggestions: [...offices.slice(1)],
+  },
+];
+
+const getSuggestions = (value, hanzi) => {
+  const inputValue = value.trim().toLowerCase();
+  const inputLength = inputValue.length;
+
+  const data = hanzi ? dataHanzi : offices;
+
+  return inputLength === 0
+    ? []
+    : data.filter(
+        (office) => office.name.toLowerCase().indexOf(inputValue) !== -1,
+      );
+};
+
+const getSuggestionValue = (suggestion) =>
+  `${suggestion.name} (${suggestion.code})`;
+
+type State = {
+  value: string,
+  suggestions: Array<any>,
+};
+
+type Props = {
+  hanzi: boolean,
+  includeIcon: boolean,
+  includeSubheading: boolean,
+  includeTertiaryLabel: boolean,
+  isBanana: boolean,
```

Comment:
We can't have a business specific property in our OSS component so this will need to be removed.

This also doesn't align with Backpack standards as this now makes it business logic and specific usecase which neither of these are to be present in Backpack due to the nature and reusability of components